session/pessimistic_test.go

```diff
@@ -817,4 +818,27 @@ func (s *testPessimisticSuite) TestPessimisticReadCommitted(c *C) {
 	wg.Wait()
 
 	tk1.MustExec("commit;")
+
+	// test RC
+	tk.MustExec("set tx_isolation = 'READ-COMMITTED'")
```

Comment:
Better to add case cover all the related executor builder path(`pointget`, `indexreader` etc). Do we need to add some `SET TRANSACTION ISOLATION` cases ?

Reply:
Yes, this case comes from a bug fix. I'm searching for more cases, any suggestions? e.g. mysql-test?

Reply:
@jackysp  mysql does have a few like `suite/innodb/t/innodb-isolation.test` , we could port some `RC` and `RR` related ones

Reply:
got